scripts/p2p-test-tool.py

```diff
@@ -28,7 +28,7 @@
 parser.add_argument("--bootstrap-command",
                     dest='bootstrap_command',
                     type=str,
-                    default="run --port 40400 --standalone",
+                    default="run --protocol-port 40400 --standalone",
```

Comment:
Just want to say that I don't think we are still using this file.

Reply:
Do you think we should remove it? Looks like there is some useful code there. I don't want to hoard stuff but it's in `script` folder so does not hurt imo.